service/common/src/main/java/org/apache/polaris/service/persistence/postgres/dao/PostgresNamespaceDao.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.persistence.postgres.dao;
+
+import java.util.List;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.polaris.core.entity.NamespaceEntity;
+import org.apache.polaris.core.entity.PolarisEntity;
+import org.apache.polaris.core.entity.PolarisEntityCore;
+import org.apache.polaris.core.persistence.dao.NamespaceDao;
+
+public class PostgresNamespaceDao implements NamespaceDao {
```

Comment:
I would rather consider to have a `PostgreSQLPersistenceAdapter` using DAO as records more than DAO implementation per backend.

Reply:
I don't quite follow here. `NamespaceDao` here is an interface with operations, not a record.

Reply:
Let me rephrase because I wasn't clear indeed:
- `PostgresNamespaceDao` or `JdbcNamespaceDao` are good names
- My point is more for the "record" used in the DAO (at interface level).